api/src/Entity/Role.php

```diff
@@ -46,6 +46,9 @@ class Role
     #[ORM\ManyToMany(targetEntity: Category::class, inversedBy: 'roles')]
     private Collection $categories;
 
+    #[ORM\Column(type: 'json')]
+    private ?array $Features = [];
```

Comment:
```suggestion
    private array $Features = [];
```
Pas besoin du ? puisqu'on déclare un array